_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
-    {%- for href, id, caption in items %}
+    {%- for item in items %}
```

Comment:
This decision probably shouldn't be limited to two scoops of FEWD. @schaferjh, any opinion?